### ISSUE TITLE
[📝 Docs]: Create a standalone introductory guide to JSON Schema

### ISSUE THREAD

**Issue by valeriahhdez** [json-schema-org/website#1129]
### What Docs changes are you proposing?

Create a self-contained document product that introduces beginners to JSON Schema to replace two documents that are currently formatted as book chapters: 

- [What is a schema?](https://json-schema.org/understanding-json-schema/about)
- [The basics](https://json-schema.org/understanding-json-schema/basics)

A book chapter format presents information in a sequential manner, where each chapter builds upon the previous one. This means chapters lack clear learning goals and the reader needs to complete more than one chapter to complete a task or learning lesson. 

An introductory guide to JSON Schema that is self-contained  should comply with the following requirements to replace the book chapters mentioned above:

- clear learning goals for the reader
- a start and end points
- link to other learning resources
- be clear, concise, and accurate

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by valeriahhdez** [comment]
Hi @techmannih, yes, you can go work on this issue

**Comment by benjagm** [comment]
Team, remember we already have this page: https://json-schema.org/overview/what-is-jsonschema

@valeriahhdez @kwennB

**Comment by valeriahhdez** [comment]
> Team, remember we already have this page: https://json-schema.org/overview/what-is-jsonschema
> 
> @valeriahhdez @kwennB

Yes, indeed. I had previously proposed to remove from the Reference content bucket these two documents precisely because we already have "What is JSON Schema?" in the Introduction content bucket but you said you preferred to keep them and that is what we did. However, "The basics" and "What is a schema?" are formatted as book chapters, which means the reader has to consult more than one document to complete a learning lesson. This is why I created this issue. Documentation should be self-contained in the sense that a document should deliver all the necessary information and context for a reader to understand a specific topic or concept without depending too much on other sources to complete a learning lesson or task.

**Comment by benjagm** [comment]
Reading this issue again, in the requirements, we are saying 

> An introductory guide to JSON Schema that is self-contained should comply with the following requirements to replace the book chapters mentioned above

However, this document shouldn't be a Guide; this is the reference section, and References have a totally different structure than guides.  So the resulting document is a reference doc (not a guide) and the goals are the same:

- clear learning goals for the reader
- a start and end points
- link to other learning resources
- be clear, concise, and accurate

**Comment by benjagm** [comment]
After long consideration, I think the current status of the site doesn't require this change. Thanks a lot for your resilience and commitment. Next time we'll do a better job during the triage.

Thanks everyone involved in the discussion.
